packages/chat-component/tsconfig.json

```diff
@@ -21,13 +21,15 @@
     "experimentalDecorators": true,
     "forceConsistentCasingInFileNames": true,
     "noImplicitOverride": true,
+    "allowImportingTsExtensions": true,
```

Comment:
Is is necessary? I think if we target a ESM bundle we should always import `.js` extension as the eslint rules enforces

Reply:
Only because we are decoupling the text processing to separate module. Let me test it and see if I can revert.